tools_for_todoist/services/calendar_to_todoist.py

```diff
@@ -90,19 +92,39 @@ def _process_cancelled_event(self, calendar_event):
             todoist_item = self.todoist.get_item_by_id(todoist_id)
             if todoist_item is not None:
                 self.todoist.delete_item(todoist_item)
+    
+    def _process_updated_event(self, calendar_event):
+        print('Updating event|', calendar_event)
+        todoist_item = self.todoist.get_item_by_id(_todoist_id(calendar_event))
+        if todoist_item is None:
+            print('Invalid event??')
+            return None
+        next_occurence = calendar_event.next_occurrence()
+        if next_occurence is None:
+            self.todoist.delete_item(todoist_item)
+            return None
+        todoist_item.set_due(next_occurence, calendar_event.recurrence_string())
+        todoist_item.save()
```

Comment:
if next due date was the same, we are still going to make an update, please change it so we don't waste time running update commands that don't do anything.
1) Either make the check here and don't execute the code if it's not necessary or
2) Make `move_due` and `save` do nothing if due date was passed as the same as current one